### DISCUSSION TITLE
Triggers for getting S3 data

### DISCUSSION THREAD

**Discussion by Philipp12343** [Budibase/budibase#8535] [Help]
Hey,

I am completly new to this, so dont expect much from me. About AWS S3 I added .json data. About which trigger can I call this data in the automation?

Best regards

Philipp

**Comment by melohagan** [comment]
Hey @Philipp12343 

I don't believe it is currently possible to read JSON content from an S3 bucket directly. You may need to add a Lambda function that reads the file and returns the content in an API response.

Then you can add a [REST query](https://docs.budibase.com/docs/rest) and use the **External Data Connector** action in automations to use that REST query.
